packages/docusaurus-1.x/lib/static/css/main.css

```diff
@@ -653,6 +653,16 @@ blockquote {
   padding: 0;
 }
 
+.mainContainer .wrapper .post .postHeader:before,
+.mainContainer .wrapper .post .postHeaderTitle:before {
```

Comment:
This caused a CSS regression  bug in which Blog post "Read more" doesnt work anymore

https://deploy-preview-1869--docusaurus-preview.netlify.com/blog/

Reply:
Thank you @yangshun for the quick fix